homeassistant/components/venstar/sensor.py

```diff
@@ -64,6 +64,8 @@
     255: "inactive",
 }
 
+STAGES: dict[int, str] = {0: "idle", 1: "first_stage", 2: "second_stage"}
```

Comment:
just an idea: what do you think about using an `IntEnum` instead?

Reply:
Most other integrations I see just use int dicts, so this sticks with the status quo.